spot_wrapper/wrapper.py

```diff
@@ -473,7 +473,10 @@ def _start_query(self) -> None:
                 self._logger.error("Error when getting robot command feedback: %s", e)
                 self._spot_wrapper._last_trajectory_command = None
 
-        self._spot_wrapper.is_moving = is_moving
+        if self._spot_wrapper._last_navigate_to_command != None:
+            is_moving = True
+
+        self._spot_wrapper._is_moving = is_moving
```

Comment:
```suggestion
        self._spot_wrapper.is_moving = is_moving
```

Reply:
This changed in https://github.com/bdaiinstitute/spot_wrapper/commit/3960dff13baba86539dbfc809c5497ff78468d82

Reply:
Fixed, thanks.